### PR TITLE
feat: add lend ui

### PR DESCRIPTION
**Pull request by karlpawlowicz**
## Purpose

1. Added a `LendCheckout` component
2. Added token images to prices

## Testing Instructions

1. Confirm the lend flow looks correct (optional):

- Pull down the branch
- Move into the `client` directory and run `npm run dev`
- Navigate to `http://localhost:3000/lend`
- Click on This would be your NFT (click here)

2. Review code

## Checklist

- [x] Title follows the [naming convention](../README.md#pull-requests).
- [x] Add a description.
- [ ] Add screenshots (optional).
- [x] Add testing instructions.
- [x] Add at least one reviewer.
- [x] Apply related labels (optional).
- [x] Leave comments on changes you would like to discuss.
- [x] Remove changes that are not related to this pull request.
- [x] Remove debugging code.
- [x] Test your changes locally and on staging (if applicable).
- [ ] Ensure test coverage value in config matches actual coverage (if applicable).
- [ ] Ensure the build is passing.

### CODE REVIEW

File: client/src/pages/RentCheckout.jsx [hunk 1]
```diff
@@ -0,0 +1,239 @@
...
+    { name: '2', available: true },
+    { name: '3', available: true },
+    { name: '5', available: true },
+    { name: '7', available: true },
+    { name: '14', available: true },
+    { name: '21', available: true },
+    { name: '28', available: true },
+]
+
+function classNames(...classes) {
```
Comment (mattxyzeth) [inline client/src/pages/RentCheckout.jsx:31]: I installed the `classnames` package if you want to use that.

File: client/src/pages/RentCheckout.jsx [hunk 2]
```diff
@@ -0,0 +1,239 @@
...
+    return classes.filter(Boolean).join(' ')
+}
+
+export default function RentCheckout() {
+    const [selectedDay, setSelectedDay] = useState(days[2])
+
+    return (
+        <div className="bg-white">
```
Comment (mattxyzeth) [inline client/src/pages/RentCheckout.jsx:39]: Does this page need the `AuthRoute` wrapper component?

**Review by mattxyzeth — APPROVED** [review]

File: client/src/pages/RentCheckout.jsx (hunk 1 above)
Comment (karlpawlowicz) [reply client/src/pages/RentCheckout.jsx:31]: I can refactor that in my next PR

File: client/src/pages/RentCheckout.jsx (hunk 2 above)
Comment (karlpawlowicz) [reply client/src/pages/RentCheckout.jsx:39]: Yes, it will need it, but I think we can add it later

**Pull request merged**